ccf_transaction_processor/etc/constitution.js

```diff
@@ -0,0 +1,824 @@
+// Default Constitution for Governance of Microsoft CCF App. The module is a concatenation of 
```

Comment:
What is the license for this? Are we allowed to use it?

Reply:
thank you, yes, CCF repo is APL 2.0. CCF itself apps to start with their default constitution and modify as per needs of the app. We are currently using the defaults without any change. Good for research prototypes.